staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -275,7 +297,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 		case <-listCh:
 		}
 		if err != nil {
-			return fmt.Errorf("%s: Failed to list %v: %v", r.name, r.expectedTypeName, err)
+			return fmt.Errorf("Failed to list %v: %v", r.expectedTypeName, err)
```

Comment:
lower case error strings here and bellow.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
hmmmm...I didn't add this error string in this PR. Would you accept this as a separate PR? This PR has been open for a couple months and has been more controversial than I expected, which makes me nervous about widening the scope to include unrelated changes.

Reply:
According to my diff, you removed the initial %s which printed a name? So I think this is a fair request. Below errors, too. We're currently in code freeze at the moment, so please expect you'll end up rebasing one or more times before this merges anyway.

Reply:
(And I see this PR had been open a long time before I got to it. Sorry about that.)

Reply:
makes sense to me, done!